src/components/GeoMap.js

```diff
@@ -423,9 +423,9 @@ export default class GeoMap {
 
       dataset.forEach(d => {
         d.rollupValue[attrKey] = attrValue.rollup(d.values);
-      });
+    });
 
-      const scaleDomain = d3.extent(dataset, d => d.rollupValue[attrKey]);
+      const scaleDomain = d3.extent(dataset, d => Number(d.rollupValue[attrKey]));
```

Comment:
Hmm - we should check against master branch to see if it's a regression or not.  I'll check and report back!